### ISSUE TITLE
Included AviSynth.dll not detected with portable settings

### ISSUE THREAD

**Issue by vonH** [staxrip/staxrip#321]
**Describe the bug**
When first running 2.1.3.0 and checking apps, the included AviSynth.dll is not detected and it prompts the user to install it (via opening a link to the Github page for the project). This seems counterproductive when the correct version (3.6.0) dll is included in the package.

**Expected behaviour**
I would expect the included version to be automatically selected and marked as 'OK' in the apps overview, with an option for the user to use an installed version.

**How to reproduce the issue**
- Download latest version
- Extract
- Open dir
- Start Staxrip.exe
- Choose 'portable' option (Settings directory in Staxrip dir)
- Check apps (F9)
- AviSynth shows as not being installed, or defaults to using installed version (in my case, this was an old version for Staxrip 2.0.8.0.

**Additional information**
I can't really provide any more information other than opinion. When the user first opens Staxrip and the option of where to save Settings is offered, if the local directory is chosen, this should automatically use the included AviSynth.dll file (as alluded to in the docs: https://staxrip.readthedocs.io/introduction.html).

I hope I've provided enough information. If the behaviour is intentional, I would appreciate a brief explanation as it seems counterproductive to included the dlls but not use them by default.

**Comment by Dendraspis** [comment]
Could be a problem with the soft link creation. :thinking:

Have you tried the latest beta?
Have you tried starting StaxRip as Admin once?

**Comment by JJKylee** [comment]
This issue is fixed in 2.1.4.4 Beta:

![StaxRip-2 1 4 4-Beta_portable-AviSynth-dll_recognition_20200906](https://user-images.githubusercontent.com/3157518/92333492-bc87b080-f03a-11ea-8479-21c564ff0254.jpg)

**Comment by vonH** [comment]
Thank you, @Dendraspis and @JJKylee. This issue has persisted throughout versions but will try the beta. Do both of you run Staxrip with admin privileges at all times?

**Comment by JJKylee** [comment]
> Do both of you run Staxrip with admin privileges at all times?

I use installed versions of AviSynth/VapourSynth for my main StaxRip app.

FYI, you need to run StaxRip with admin privileges **just one time**. That's for creating soft links for portable frameservers. You can run StaxRip as a normal user afterwards.

Or if you don't like this process, you can turn the **Developer Mode** on in Windows Settings.

**Comment by vonH** [comment]
> > Do both of you run Staxrip with admin privileges at all times?
> 
> I use installed versions of AviSynth/VapourSynth for my main StaxRip app.
> 
> FYI, you need to run StaxRip with admin privileges **just one time**. That's for creating soft links for portable frameservers. You can run StaxRip as a normal user afterwards.
> 
> Or if you don't like this process, you can turn the **Developer Mode** on in Windows Settings.

I didn't know it needed to be run once in order to create the links. I swear I didn't see that in the documentation, but that explains a lot! Thank you. 

I would like to make a suggestion: add a warning on first run, if possible. I'll close this as it's resolved (not an error but intention design/user error)

**Comment by JJKylee** [comment]
> I swear I didn't see that in the documentation, but that explains a lot! Thank you.

Well, it's explained in one of the two StaxRip related threads in Doom9: **[here](https://forum.doom9.org/showthread.php?p=1916847#post1916847)**.

Yes, I admit it's kind of hard to follow all the posts on the threads.

@stax76 , I guess you better include this "workaround solution" in the FAQ section of documentation.
